src/api/app/views/webui2/webui/repositories/distributions.html.haml

```diff
@@ -7,8 +7,8 @@
     %h3.mb-3 Add Repositories to #{@project}
```

Comment:
I suggest we remove the `.mb-3` class since the images are now a bit taller (from 26px to 40px) and the spacing between this header and the header of the first distribution is already good with the `.pt-3` class.

Reply:
Yup, seems fine to me